CONTRIBUTING.md

```diff
@@ -151,6 +151,36 @@ Running the script for a project in `sdk\tables` would look like this:
 eng\scripts\Export-API.ps1 tables
 ```
 
+## Updating Sample Snippets
+If the specific client library has sample snippets in markdown format, they were most likely created with help of the `eng\scripts\Update-Snippets.ps1` script.
+Any changes made to the snippet markdown should be done via updating the corresponding C# snippet code and subsequently running the script.
```

Comment:
Should we discuss convention for where you would normally find these, or how I could locate them?

Reply:
I feel like this is mostly covered by the directly linked examples below and the reference to the samples guidelines. Do you think being more explicit with a dir structure example here would help make it more clear?

Reply:
It's a fair point.  I can see someone clicking the sample and looking at the resulting code but not thinking to pay attention to the path, but I could just be overthinking it.